src/libsystemd/sd-journal/journal-def.h

```diff
@@ -47,11 +47,12 @@ typedef enum ObjectType {
 
```

Comment:
Is not indexing the message such a limitation though? I might be wrong, but doing exact matches on the MESSAGE field seems like a very rare use case. Instead, I'd expect most querying of MESSAGE fields to happen via the --grep option instead. Am I wrong here?

Reply:
i actually use this all the time. i.e. add a marker somehwere with a simple `log_notice("wuff")` then search for it with `journalctl _MESSAGE=wuff`.

Reply:
Hmm, I don't mind keeping the default to index everything (which we probably should do regardless to keep compat intact) but it'd be nice to have the environment knob available to disable indexing for fields since all this indexing information does end up taking quite a bit of space.